### PR TITLE
Field names

### PR DESCRIPTION
**Pull request by rofrankel**
Adopt AIP-140: Field names

## 🍱 Types of changes

What types of changes does your code introduce to AEP? _Put an `x` in the boxes
that apply_

- [ ] Enhancement
- [ ] [New proposal](https://aep.dev/1#workflow)
- [x] Migrated from google.aip.dev
- [ ] Chore / Quick Fix

## 📋 Your checklist for this pull request

Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for
contributing to this repository.

### General

- [x] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
- [x] Ensure that your PR
      [references AEPs](https://aep.dev/style-guide#referencing-aeps)
      correctly.
- [x] [My code has been formatted](https://aep.dev/contributing#formatting)
      (usually `prettier -w .`)

### CODE REVIEW

File: aep/general/0140/aep.md.j2 [hunk 1]
```diff
@@ -1,5 +1,165 @@
...
+including adjectives that always apply and add little cognitive value. For
+example, a `proxy_settings` field might be as helpful as
+`shared_proxy_settings` if there is no unshared variant.
+
+**Important:** Field names often appear in generated client surfaces. Ensure
+they are appropriately descriptive and of suitable length.
+
+### Case
+
+{% tab proto %} Field definitions in protobuf files **must** use
```
Comment (rambleraptor) [inline aep/general/0140/aep.md.j2:32]: nit: newline after tab proto tag

File: aep/general/0140/aep.md.j2 [hunk 2]
```diff
@@ -1,5 +1,165 @@
...
+APIs **should** endeavor to use the same name for the same concept and
+different names for different concepts wherever possible. This includes names
+across multiple APIs, in particular if those APIs are likely to be used
+together.
+
+### Arrays / Repeated fields
+
+Arrays (in OAS) and repeated fields (in protobuf) **must** use the proper
+plural form, such as `books` or `authors`. On the other hand, singular fields
+**should** use the singular form such as `book` or `author`.
```
Comment (rambleraptor) [inline aep/general/0140/aep.md.j2:62]: Upgrade to must?

(A singular field called "books" feels unintuitive to me)

File: aep/general/0140/aep.md.j2 [hunk 3]
```diff
@@ -1,5 +1,165 @@
...
+
+For uniformity, field names that contain both a noun and an adjective
+**should** place the adjective _before_ the noun. For example:
+
+- `collected_items` (**not** `items_collected`)
+- `imported_objects` (**not** `objects_imported`)
+
+### Verbs
+
+Field names **must not** be named to reflect an intent or action. They **must
```
Comment (rambleraptor) [inline aep/general/0140/aep.md.j2:95]: Formatting nit: Put the part starting with "Rather" in a separate block (maybe a Note block?)

**Review by rambleraptor — APPROVED** [review]

File: aep/general/0140/aep.md.j2 (hunk 1 above)
Comment (rofrankel) [reply aep/general/0140/aep.md.j2:32]: Done.

File: aep/general/0140/aep.md.j2 (hunk 2 above)
Comment (rofrankel) [reply aep/general/0140/aep.md.j2:62]: Done.

File: aep/general/0140/aep.md.j2 (hunk 3 above)
Comment (rofrankel) [reply aep/general/0140/aep.md.j2:95]: Not sure I agree with this one - I think it's a continuation of the same thought.  I put it in a new paragraph to err on the side of reviewer feedback, but I don't think it makes sense in a Note: block.

**Pull request merged**